lib/matplotlib/backends/backend_gdk.py

```diff
@@ -441,6 +441,12 @@ def _renderer_init(self):
         self._renderer = RendererGDK (gtk.DrawingArea(), self.figure.dpi)
 
     def _render_figure(self, pixmap, width, height):
+        if isinstance(self._renderer, RendererGDK):
+            warn_deprecated('2.0', message="The GDK backend is "
+                            "deprecated. It is untested, known to be "
+                            "broken and will be removed in Matplotlib 2.2. "
+                            "Use the Agg backend instead.",
```

Comment:
I thought so too but I don't see it without explicitly adding the last line.

This is the warning message from the above as it looks for me
```
/usr/local/lib/python2.7/site-packages/matplotlib/cbook.py:137: MatplotlibDeprecationWarning: The GTK backend is deprecated. It is untested, known to be broken and will be removed in Matplotlib 2.2. Use the GTKAgg backend instead.
  warnings.warn(message, mplDeprecation, stacklevel=1)
```